packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -278,6 +279,11 @@ class WebView extends StatefulWidget {
   /// By default `debuggingEnabled` is false.
   final bool debuggingEnabled;
 
+  /// Controls whether WebView have bounce effect in iOS
+  ///
+  /// By default `scrollViewBounces` is true.
+  final bool scrollViewBounces;
```

Comment:
I'm wondering whether it makes sense to phrase this as a platform-agnostic API?
e.g I'd imagine that on Android an equivalent knob will control whether the overflow glow is showing or not (not sure whether the Android webview currently allows such control).

In that case maybe it makes sense to name this something like `allowOverScroll` ?